data-raw/colombia_codes.R

```diff
@@ -22,7 +22,11 @@ level_1_region <- read_html(co_iso) %>%
   html_text()
 level_1_region <- level_1_region[1:33]
 
-colombia_codes <- data.frame(level_1_region_code, level_1_region) %>%
+colombia_codes <- data.frame(
```

Comment:
chuckle. Might be a good idea to use tibble here as better defaults (i.e this default)